src/assets/style/airport.css

```diff
@@ -65,22 +65,30 @@
   width: 5em;
   font-weight: bold;
   font-size: 120%;
-  margin-right: 0.25em;
   text-align: right;
   white-space: nowrap;
 }
 
 #airport-switch li .icao {
   display: inline-block;
-  width: 4em;
-  /*margin-right: */
+  width: 2.5em;
+  margin-left: 1em;
+  text-align: center;
+  font-weight: bold;
+}
+
+#airport-switch li .symbol {
+  display: inline-block;
```

Comment:
ah! perfect. I tried to conform with what I was seeing elsewhere, but there were no exact examples so I figured I had it wrong. Will read the linked document when I get bored :smiley: